tests/scripts/all.sh

```diff
@@ -2499,13 +2499,10 @@ component_build_armcc () {
     armc6_build_test "--target=aarch64-arm-none-eabi -march=armv8.2-a"
 }
 
-component_test_allow_sha1 () {
-    msg "build: allow SHA1 in certificates by default"
-    scripts/config.py set MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_CERTIFICATES
-    make CFLAGS='-Werror -Wall -Wextra'
-    msg "test: allow SHA1 in certificates by default"
-    make test
-    if_build_succeeded tests/ssl-opt.sh -f SHA-1
+component_build_ssl_hw_record_accel() {
```

Comment:
Rebase mistake: `component_build_ssl_hw_record_accel` is being re-added.

Reply:
rebased again. thanks for spotting this